src/pages/pages-styles.css

```diff
@@ -0,0 +1,31 @@
+.page-container {
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.image-container {
+  width: 40%;
+  display: flex;
+  align-items: center;
```

Comment:
I think this ``align-items: center;`` is not being used. 🤔

Reply:
Wow, you're right! Thanks for noticing!